client/state/selectors/can-site-view-atomic-hosting.js

```diff
@@ -31,17 +24,5 @@ export default function canSiteViewAtomicHosting( state ) {
 		return false;
 	}
 
-	const canManageOptions = canCurrentUser( state, siteId, 'manage_options' );
-	if ( ! canManageOptions ) {
-		return false;
-	}
-
-	const isAtomicSite = !! isSiteAutomatedTransfer( state, siteId );
```

Comment:
And since we're showing for all sites that are or can be Atomic, just skip all the additional checks.

Reply:
Clever 👏